modules/tobira/src/main/java/org/opencastproject/tobira/impl/TobiraApi.java

```diff
@@ -82,6 +102,26 @@ public void setWorkspace(Workspace workspace) {
     this.workspace = workspace;
   }
 
+  @GET
+  @Path("/version")
+  @Produces(APPLICATION_JSON)
+  @RestQuery(
+      name = "version",
+      description = "The Tobira Module API version",
+      restParameters = {},
+      responses = {
+          @RestResponse(description = "Version information", responseCode = HttpServletResponse.SC_OK)
+      },
+      returnDescription = "JSON object with string field 'version'"
+  )
+  public Response version() {
+    var body = Jsons.obj(Jsons.p("version", VERSION));
+    return Response.ok()
+        .type(APPLICATION_JSON_TYPE)
```

Comment:
Is this necessary when you already have the `@Produces` annotation?

Reply:
You're right! https://docs.oracle.com/javaee/7/api/javax/ws/rs/Produces.html

> A method for which there is a single-valued `@Produces` is not required to set the media type of representations that it produces: the container will use the value of the `@Produces` when sending a response.